README.md

```diff
@@ -14,7 +14,7 @@ We will open a browser using LiveKit's [meet](https://meet.livekit.io) product a
 - Run `pip install -r requirements.txt` in your environment (e.g. conda, venv) of choice.
 
 ## Running It
-Run `python stream_script.py` to start a LiveKit room and get started with object tracking. If you included a token in `LIVEKIT_TOKEN_FOR_RESULTS`, you'll also see 
+Run `python stream_script.py` to start a LiveKit room and get started with object tracking. If you included a token in `LIVEKIT_TOKEN_FOR_RESULTS`, you'll also see real-time object tracking results returned to the client via WebRTC data channels, along with a measure of latency from DirectAI's backend receiving the frame from LiveKit to the client receiving the tracked object results. We would expect this latency to be <200ms.
```

Comment:
nit: second sentence is run-on

suggested rephrasing: If you include a token in `LIVEKIT_TOKEN_FOR_RESULTS`, you'll see real-time object tracking results and end-to-end latency measurements returned to the client via WebRTC data channels.

Reply:
makes sense re: distinction with e2e latency

a couple notes on the revised version:
- I'd prefer "If you include" as opposed to "If you included" as it's a present/future instruction
- Where is the measure of latency being sent to? I assume the webrtc data channel but it's not obvious from my reading

Reply:
it's not being sent anywhere, it's being measured and printed in this code